docs/debug+profile/clang/blacklists.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Blacklists (Clang Sanitisers)
+---
+
+A blacklist file contains the names of source files and/or functions whose issues (races, leaks, ...) should not be reported. Such blacklist files can be placed anywhere on the system. For the sake of simplicity, this article will focus on working with the [ThreadSanitizer](/docs/debug+profile/clang/threadsanitizer/). However, the same concepts and rules apply for all [Clang sanitisers](/docs/debug+profile/clang/) when used in combination with Mono.
+
+The simplest approach (an empty blacklist) would look like this:
+
+``` bash
+$ touch /home/some/path/blacklist
+$ cd /home/root/of/mono
+$ ./autogen.sh --prefix=/usr/local CC=clang CFLAGS="-fsanitize=thread -fsanitize-blacklist=/home/some/path/blacklist" LDFLAGS=-fsanitize=thread CXX=clang++
+$ make
+...
+```
+
+Currently (August 2017), it seems like a good idea to leave all blacklist files outside of folders that contain `Makefile` files (`mono/eglib/`, `mono/metadata/`, ...), as sometimes `autogen.sh` treats blacklist files like source files and lists them as dependencies. As an additional and positive side effect, blacklist files will not interfere with git if they are hosted outside of the repository.
```

Comment:
I didn't change that part much (had it there with the first version of the TSan article already); it was interesting but running `./autogen` and `make` consecutively with and without TSan options and having the blacklist somewhere inside the `mono/` folder (and not) gave more trouble (something about missing files) than having it somewhere outside (somehow it seemed to be connected to the built libraries inside the `./libs` folders). We can remove this part however, until I know exactly what's going on there.